packages/block-editor/src/components/link-control/test/fixtures/index.js

```diff
@@ -1,7 +1,6 @@
-/**
- * External dependencies
- */
-import { uniqueId } from 'lodash';
+export const uniqueId = () =>
```

Comment:
I don't know if the replacement function is needed - the usage of `uniqueId` in these tests could just be replaced with hard-coded id values.

lodash `uniqueId` is just a counter that counts up from `1` rather than a random number:
![Screen Shot 2022-06-20 at 2 44 22 pm](https://user-images.githubusercontent.com/677833/174541153-9f043489-db04-4629-9f45-b1b5fddae972.png)

So I'd expect every test run to produced the same ids, the first call to `uniqueId()` would return '1', the second '2' etc.

Reply:
Good point. Happy to simplify 👍 Done in https://github.com/WordPress/gutenberg/pull/41751/commits/8c4d4863abedfcadd5ce3d10d1a227ae537fde01